isis/src/base/objs/StringBlob/StringBlob.cpp

```diff
@@ -7,82 +7,55 @@ find files of those names at the top level of this repository. **/
 /* SPDX-License-Identifier: CC0-1.0 */
 #include <string>
 #include "StringBlob.h"
-#include "Application.h"
+#include "Blob.h"
 
 using namespace std;
 namespace Isis {
   /**
    * Constructor for creating a string blob with no arguments
    */
-  StringBlob::StringBlob() : Isis::Blob("IsisCube", "String") {
+  StringBlob::StringBlob() {
     m_string = "";
+    m_name = "";
```

Comment:
I don't know if we want to carry around the name of the PVL container name for the Blob PVL. Similarly to the history object, this should probably only live in the IsisCube